### ISSUE TITLE
1.0.4 rejected by Play Store

### ISSUE THREAD

**Issue by justdave** [justdave/MCStatus#22]
1.0.4 also got rejected by the Play Store.
I have no idea what's left that's violating the developer policy.  They won't tell me which policy it's violating, just "here's the policy, make sure it complies", and I can't find anything in it that isn't in compliance.  If anyone's got ideas I'd love to hear them.

-----
After a recent review, we found that your app MCStatus (net.justdave.mcstatus) is not compliant with one or more of our Developer Program Policies. See below for more information about your app’s status and how to correct the issue.

### About the Google Play policy
Google reserves the discretion to include or remove apps from Google Play. We may take action based on a number of factors including, but not limited to, a pattern of harmful behavior or high risk of abuse. We identify risk of abuse using various items such as previous violation history, user feedback, and use of popular brands, characters, and other assets.
### App status: Rejected
Your app has been rejected and wasn't published due to a policy violation. If you submitted an update, the previous version of your app is still available on Google Play.

### Action required: Submit an updated app for review
Here’s what to do to help get your app on Google Play:
1. 	Read through the [Policy Coverage page](https://play.google.com/about/enforcement/policy-coverage/) in the Developer Policy Center. 
2. 	Make appropriate changes to your app, and be sure to address the issue described above. Also check your app’s store listing for compliance, if applicable.
3. 	Double check that your app is compliant with all other [Developer Program Policies](https://play.google.com/about/developer-content-policy/). 
4. 	Sign in to your [Play Console](https://play.google.com/console/) and submit the update to your app.

**Comment by justdave** [comment]
Followup from the appeal:

-----
During review, we found that your app violates Google Play policy, and determined your app to be a high risk profile. Google Play identifies high risk profiles by using signals such as:

* apps that contain assets prone to abuse

You can read through the Policy Coverage page for more details.

For example, your app currently contains **content of a popular brand.**

In order for us to process your appeal, please reply with verifiable documentation that your app is authorized to use this content. Examples of verifiable documentation(PDF) include, but not limited to: distribution agreement, authorization contract, website domain information and et cetera.

If you are not authorized to use this content, please remove any content of the popular brand.

-----

The only thing in the app that seems remotely close to that is the use of the Minecraft name.

I replied with a link to Mojang's Brand and Asset Usage Guidelines at https://account.mojang.com/terms#brand which we appear to be in compliance with as far as I can tell.

**Comment by justdave** [comment]
I just committed an update to add a link to the Mojang trademark license agreement into the about box (2a2e53e52a0a1767c0d98ecd19ba971fa49ad4f5).  I'll submit that and see if it placates them any.

**Comment by justdave** [comment]
Success! The release is live.  That sure was an adventure.

**Comment by Mindavi** [comment]
Good job! Nice to see the greatly improved version going live. I sure missed some things, but it was quite a while since I did android stuff and I wasn't too aware of all the nitty-gritty backwards compat stuff that is involved in building apps. Great work updating all of it and it all seems to work a lot better now. Thanks for the hard work!

**Comment by justdave** [comment]
Thanks for your pull request that got this all started. :-)  I hadn't touched Android stuff in quite a while either, I spent a day or two just trying to get my development environment working again after I saw your PR. The backward-compat stuff I only noticed because Android Studio complained about it when I merged your PR.  It was good timing anyway because in testing it I discovered they changed stuff in Minecraft 1.15 that broke the app, and it needed updating to handle 1.15 servers.  Feel free to do it again if you notice anything you want to tackle!
